lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -46,10 +46,20 @@ def save_to_file(filename, content)
             "cat << EOF > #{filename}\n#{content}EOF"
           end
 
-          def indent(count)
+          def indent(count, skip1: false)
```

Comment:
Probably `skip_first_line` would be more descriptive, than `skip1`.

Reply:
Yeah that was my initial idea but feels quite long. If there is one more observation I will rebase ;-)

Reply:
Ok, since it's already used in templates, I will not insist anymore